controller/src/main/java/io/pravega/controller/store/stream/PravegaTablesStream.java

```diff
@@ -278,15 +277,32 @@ private String getWritersTableName(String id) {
     }
 
     @Override
-    public CompletableFuture<Void> createSubscriber(String newSubscriber) {
+    public CompletableFuture<Void> createSubscriber(String newSubscriber, long operationGeneration) {
```

Comment:
we are setting generation in create but we need to do the same in update subscriber